src/java/org/apache/cassandra/service/accord/txn/TxnUpdate.java

```diff
@@ -185,6 +178,7 @@ public Write apply(Data data)
         return new TxnWrite(updates);
     }
 
+    // Should we serialize the conditionResult?
```

Comment:
If we serialize the result, does that also remove the need for the `checkCondition()` method below? (i.e. The condition should already have been applied somewhere and populated...and `null` would mean something went wrong?) If so, +1

Reply:
The conclusion here was that we don't need to. At the coordinator it will not be recomputed in memory and on recovery it is safe for a recovery coordinator to recompute it.